### ISSUE TITLE
support back option in import wizard

### ISSUE THREAD

**Issue by mariodavid** [mariodavid/cuba-component-data-import#71]
```
As a user of the import wizard
When I have made a mistake in a previous step
Then I would like to have the ability to go one step back in the wizard so that I can correct my mistake
```

**Comment by innovaringenieria** [comment]
Is it possible to add the way to continue importing files without having to exit and enter again?

**Comment by mariodavid** [comment]
Can you clarify what you mean @innovaringenieria ? In case it is not related to the description above, please feel free to create another issue that deals with your requirement. Thx

**Comment by mariodavid** [comment]
implemented in #74
